### ISSUE TITLE
Programming language logos

### ISSUE THREAD

**Issue by ioquatix** [ioquatix/programming.dojo.net.nz#20]
Ideally we update them and use SVGs where possible.

cc @courtneycb

**Comment by courtneycb** [comment]
Struggling to find SVGs for Basic, Pascal and Smalltalk.

**Comment by ioquatix** [comment]
Yeah it’s not easy! Maybe we should see if we can get a designer to make something?

**Comment by ioquatix** [comment]
@courtneycb thanks for the HUGE improvement.

<img width="717" alt="image" src="https://user-images.githubusercontent.com/30030/72864841-60d21680-3d3a-11ea-8338-e660a6125f01.png">

It's looking so much better.

The only thing I'd suggest is whether we can make the logos a bit bigger. Maybe other layout/style improvements could be great too. But, I'm happy to close this issue, let me know what you think and if you have time/energy to work on it further.

**Comment by ioquatix** [comment]
Maybe for scratch we could add the cat character?
